### ISSUE TITLE
Targeting an object changes the font size of the entire UI

### ISSUE THREAD

**Issue by nstgeorge** [nstgeorge/VEKTA#78]
### Description

When you target a planet, the entire UI's font size changes.

### System Information

This was tested on my laptop with a 3200x1800 display while the game was running in 1920x1080 windowed.

### Steps to Reproduce
1. Target a planet.
2. Untarget while watching the UI.

### Actual Results
All the text shifts around and becomes larger when the object is targeted.

### Expected Results
Everything should stay the same size.

**Comment by rvanasa** [comment]
A short-term fix could be using [pushStyle()](https://processing.org/reference/pushStyle_.html) and [popStyle()](https://processing.org/reference/popStyle_.html), although this is will no longer be an issue once we rework the overlay/menu system.

**Comment by nstgeorge** [comment]
Yep, that's what I was thinking. Not sure what the performance cost is on those, but I doubt it's expensive. I'll give it a shot tomorrow.

**Comment by rvanasa** [comment]
I'm trying to reproduce / pin down what's happening. Let me know if you figure out the cause, and I'll also look for a clean way to fix this for now. Performance should be negligible as long as it's only happening a few times per frame.

**Comment by nstgeorge** [comment]
This might be related to display density. It only seems to happen when I run the game at higher resolutions (>1080p).
